include/soci/firebird/soci-firebird.h

```diff
@@ -139,6 +145,14 @@ struct firebird_standard_use_type_backend : details::standard_use_type_backend
 
     char *buf_;
     short indISCHolder_;
+
+private:
+    // Allocate a temporary blob, fill it with the data from the provided
+    // string and copy its ID into buf_.
+    void init_as_blob(const std::string& in);
```

Comment:
Same here, `copy_as_blob`. Especially, those `init` functions are not called from constructor.